src/Talk/Create/CreateTalkRequest.php

```diff
@@ -2,43 +2,37 @@
 
 declare(strict_types=1);
 
-namespace App\Dto;
+namespace App\Talk\Create;
 
-use App\Entity\Talk;
+use App\Entity\Speaker;
+use App\Talk\Talk;
 use Symfony\Component\Validator\Constraints as Assert;
 
-final class TalkRequest
+class CreateTalkRequest
```

Comment:
Just as for the form, if anything I'd love to have an abstract class and two final descendants.

I'd still rather keep them duplicated.